src/query/settings/src/lib.rs

```diff
@@ -437,6 +437,16 @@ impl Settings {
                 desc: "How many hours will the COPY file metadata expired in the metasrv, default value: 24*7=7days",
                 possible_values: None,
             },
+            SettingValue {
+                default_value: UserSettingValue::UInt64(1),
+                user_setting: UserSetting::create(
+                    "hide_options_in_show_create_table",
```

Comment:
If it's a Boolean, the name `enable_options_in_show_create_table`(Default is false) is more better?

Reply:
`hide_options_in_show_create_table` is good to understand as well.